dkpro-core-api-lexmorph-asl/src/main/resources/de/tudarmstadt/ukp/dkpro/core/api/lexmorph/tagset/fr-sequoia-pos.map

```diff
@@ -0,0 +1,30 @@
+*=de.tudarmstadt.ukp.dkpro.core.api.lexmorph.type.pos.POS
```

Comment:
Please add at least one comment line at the beginning with a human-readable name for the tagset. Better even add more comments lines separated from the first one by an empty comment line with a human-readable description of the tagset. 

Also, please use `__META_SOURCE_URL__` and `__META_TYPE_BASE__`.

See here for an example: 

```
# MElt (CoreNLP 3.4)
#
# French POS Tagger Tagset used by Stanford CoreNLP 3.4 models
#
# Follows Benoit Crabbé and Marie Candito. 2008
#
# Benoit Crabbé and Marie Candito. 2008. Expériences d’analyse syntaxique statistique du français.
# In Proc. of TALN’08, pages 45–54, Senlis, France.
# http://hal.inria.fr/docs/00/52/57/69/PDF/crabbecandi-taln2008-final.pdf
# 
# Mapped according to universal_pos_tags.1.03 (http://code.google.com/p/universal-pos-tags/)
# This does not apply to the differences introduced in CoreNLP 3.4! They were mapped on a 
# best-effort base.

__META_SOURCE_URL__=http://opus.lingfil.uu.se/tools/public/tagger/melt-0.6/README
__META_TYPE_BASE__=de.tudarmstadt.ukp.dkpro.core.api.lexmorph.type.pos.
```